src/DNA/native/src/Generics.c

```diff
@@ -112,7 +120,7 @@ tMD_TypeDef* Generics_GetGenericTypeFromCoreType(tMD_TypeDef *pCoreType, U32 num
 	memset(pTypeDef, 0, sizeof(tMD_TypeDef));
 	// Make the name of the instantiation.
 	strcpy(name, pCoreType->name);
-	strcat(name, "[");
```

Comment:
@chrisdunelm I know, but it's very confusing to see `[` and `]` instead of `<` and `>` when types are printed out. It's also mainly cosmetic, so it shouldn't matter. And it's easy to change back if needed.